services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -159,7 +159,7 @@ open class ApplicationService(
         }
 
         val hanke = application.hanke
-        val hankeId = hanke.id!!
+        val hankeId = hanke.id
```

Comment:
Can possibly be removed.

Reply:
Thank you for noticing. I inlined the value.